website/content/en/docs/getting-started-with-kops/_index.md

```diff
@@ -0,0 +1,195 @@
+
+---
+title: "Getting Started with Karpenter using kOps"
+linkTitle: "Getting Started with kOps"
+weight: 10
+---
+
+In this example, the cluster will be running on Amazon Web Services (AWS) managed by [kOps](https://kops.sigs.k8s.io/).
+Karpenter is designed to be cloud provider agnostic, but currently only supports AWS. Contributions are welcomed.
+
+Karpenter is supported on kOps as of 1.24.0-alpha.2, but sits behind a feature flag as the interface between kOps and Karpenter is
+still a work in progress. This guide is intended for users who want to test Karpenter on kOps and provide feedback to Karpenter and kOps developers.
+Read more about how Karpenter works on kOps and the current limitations in the [kOPs Karpenter documentation](https://kops.sigs.k8s.io/operations/karpenter/).
+
+This guide should take less than 1 hour to complete, and cost less than $0.25.
+Follow the clean-up instructions to reduce any charges.
+
+This guide assumes you already have a kOps state store and a hosted zone. If you do not have one,
+run through the [kOps getting started on AWS documentation](https://kops.sigs.k8s.io/getting_started/aws/) up until "Creating your first cluster".
+
+## Install
+
+Karpenter is installed in clusters as a managed addon. kOps will automatically create 
+and manage the necessary IAM roles and policies Karpenter needs.
+
+### Required Utilities
+
+Install these tools before proceeding:
+
+1. `kubectl` - [the Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+2. `kops` - [kubectl, but for clusters](https://kops.sigs.k8s.io/getting_started/install/) v1.24.0 or later
+
+### Environment Variables
+
+After setting up the tools, set the following environment variables used by kOps.
+
+```bash
+export KOPS_FEATURE_FLAGS=Karpenter
+export CLUSTER_NAME=${USER}-karpenter-demo.example.com
+export ZONES=us-west-2a
+export KOPS_STATE_STORE=s3://prefix-example-com-state-store
+export KOPS_OIDC_STORE=s3://prefix-example-com-oidc-store/discovery
+```
+
+### Create a Cluster
+
+kOps installs Karpenter on the control plane. Once the control plane is running, Karpenter will provision
+the worker nodes needed for non-Control Plane Deployments such as CoreDNS and CSI drivers.
+
+The following command will launch a cluster with Karpenter-managed worker nodes:
+
+```bash
+kops create cluster \
+    --zones=$ZONES \
+    --discovery-store=${KOPS_OIDC_STORE} \
+    --instance-manager=karpenter \
+    --networking=amazonvpc \
+    ${CLUSTER_NAME} \
+    --yes
+```
+
+Note: we are using AWS VPC CNI for networking as Karpenter's binpacking logic assumes ENI-based networking.
+
+### Create Grafana dashboards (optional)
```

Comment:
We're planning on making a separate getting started guide for grafana. I'd leave this out of here for now.